scripts/uploadToS3.js

```diff
@@ -32,11 +32,22 @@ const files = glob.sync(options.files, {
 })
 
 const generateHeaders = file => {
-  const contentType = mime.getType(path.extname(file))
+  const extension = path.extname(file)
+  const contentType = mime.getType(extension)
+  const shouldGzip = [".js", ".css", ".gz"].some(ext => ext === extension)
```

Comment:
We should gzip a `.gz` file?

Reply:
Maybe this var should be renamed -- this just says "if gzip, add ContentEncoding: gzip" header

Reply:
ohhhh. Makes sense.